R/rdeephaven/R/client_wrapper.R

```diff
@@ -117,22 +112,20 @@ Client <- R6Class("Client",
         client_options = options
       )
     },
-    
+
     empty_table = function(size) {
       verify_nonnegative_int("size", size, TRUE)
       return(TableHandle$new(self$.internal_rcpp_object$empty_table(size)))
     },
-    
+
     time_table = function(period, start_time = "now") {
       verify_string("period", period, TRUE)
-      verify_string("start_time", start_time, TRUE)
+      if(any(start_time != "now") && any(start_time != "")) {
+        verify_string("start_time", start_time, TRUE)
+      }
```

Comment:
I don't understand this change.  Shouldn't you be verifying that it is a string no matter what?  Then the whole "now" or "" thing is not making sense to me.